### PR TITLE
Document new 1.11 and 1.10.1 configuration options

### PR DESCRIPTION
**Pull request by domcleal**
I notice the formatting is broken in this section of the manual with the default/see also lines missing line breaks.  For now I've just followed what's there.  I'll try to follow up with a fix for that across the whole section.

### CODE REVIEW

File: _includes/manuals/1.11/3.5.2_configuration_options.md [hunk 1]
```diff
@@ -390,10 +407,17 @@ Default: true
...
+##### ssl_client_cert_env
+
+Environment variable containing the entire PEM-encoded certificate from the client.  This environment variable is required when authenticating using Subject Alternative Names and will be preferred over `ssl_client_dn_env` if available.  Under Apache HTTP and mod_ssl, `SSLOptions +ExportCertData` sets this environment variable.
+Default: SSL_CLIENT_CERT
+See also: ssl_client_dn_env
+
 ##### ssl_client_dn_env
 
-Environment variable containing the subject DN from a client SSL certificate
+Environment variable containing the subject DN from a client SSL certificate.  Under Apache HTTP and mod_ssl, `SSLOptions +StdEnvVars` sets this environment variable.
```
Comment (mmoll) [inline _includes/manuals/1.11/3.5.2_configuration_options.md:418]: One space too much before the added sentence?

Comment (mmoll) [inline _includes/manuals/1.11/3.5.2_configuration_options.md:418]: scrap that, it's like this in the entire file...

**Comment by mmoll** [comment]
merged, thanks @domcleal!

File: _includes/manuals/1.11/3.5.2_configuration_options.md (hunk 1 above)
Comment (domcleal) [inline _includes/manuals/1.11/3.5.2_configuration_options.md:418]: It's a lifelong habit for me.  I'm trying to break it.

**Pull request merged**